### ISSUE TITLE
how to increase threads usage on the server

### ISSUE THREAD

**Issue by dhruvhehe** [thecoder-001/MineColab#122]
My issue is not concerned with any problem.
 i wanted to ask you how to increase threads on the server as I want to host this for approx 80 people.
is it possible to host this server smooth to 80 people?
is there any alternative or any way to do so?

**Comment by N-aksif-N** [comment]
@dhruvhehe  Nah, no. You can't because Google collab gives the fixed threads.

Well,as Google colab say, we can not check exactly the cpu which colab provides.
![image](https://github.com/user-attachments/assets/bb4b0ce0-75cb-4b4d-9de2-a6d39853b32d)
Though this can be checked by using this collab https://colab.research.google.com/drive/1dint4ly-7h8Trw0XRJ1uhC_VKe_wDJfY#scrollTo=Ov6PHWiF4QuO

A better version: https://github.com/N-aksif-N/MineColab_Improved
Join our discord for more help: https://discord.gg/F9nPJTn7Nu
